src/components/modal/Modal.js

```diff
@@ -18,26 +18,22 @@ ModalActions.defaultProps = {
   className: ''
 }
 
-export const ModalBody = ({ className, icon, title, children, ...props }) => {
-  icon = React.createElement(icon, {
-    className: 'fill-gray w3'
-  })
-
-  return (
-    <div className={`ph2 pv3 tc ${className}`} {...props}>
+export const ModalBody = ({ className, Icon, title, children, ...props }) => (
+  <div className={`ph2 pv3 tc ${className}`} {...props}>
```

Comment:
```suggestion
  <div className={`ph4 pv3 tc ${className}`} {...props}>
```
To avoid ugly margins.

Reply:
Warning: This will change all modals

Reply:
Yep, I'll check them all.